### PR TITLE
Instrument the content moderation feature

### PR DESCRIPTION
**Pull request by dmarcoux**
Nothing fancy, just the usual callbacks in models to track what happened. With this information, I plan on having panels in our Grafana instance to answer the following questions:

### Questions which can be answered with the current instrumentation
- [x] How many reports have been submitted?
- [x] Is there a certain report category which is more common than others?
- [x] Are the reports for comments, packages, projects, etc?
- [x] How long does it take for a decision to be taken after a report has been submitted?
- [x] Did the decisions favor or clear reports?

### CODE REVIEW

**Comment by dmarcoux** [comment]
[Instrument report and decisions notifications (M)](https://trello.com/c/I34Vsd0A/2351-instrument-report-and-decisions-notifications-m)

File: src/api/app/models/decision.rb [hunk 1]
```diff
@@ -24,6 +27,10 @@ def create_event
   def event_parameters
     { id: id, moderator_id: moderator.id, reason: reason, reportable_type: reports.first.reportable.class.name }
   end
+
+  def track_decision
+    RabbitmqBus.send_to_bus('metrics', "decision,kind=#{kind},hours_before_decision=#{((updated_at - created_at) / 1.hour).floor} count=1")
```
Comment (bmwiedemann) [inline src/api/app/models/decision.rb:32]: `hours_before_decision` is a tag here - and tags don't like to have a large number of different values. Better add it as a value after the space.

File: src/api/app/models/decision.rb [hunk 2]
```diff
@@ -24,6 +27,10 @@ def create_event
...
+  end
```
Comment (hennevogel) [inline src/api/app/models/decision.rb:33]:
```suggestion
  def track_decision
    hours_before_decision = ((updated_at - created_at) / 1.hour).floor
    RabbitmqBus.send_to_bus('metrics', "decision,kind=#{kind},hours_before_decision=#{hours_before_decision} count=1")
  end
```

File: src/api/app/models/report.rb [hunk 3]
```diff
@@ -37,6 +38,10 @@ def create_event
   def event_parameters
     { id: id, user_id: user_id, reportable_id: reportable_id, reportable_type: reportable_type, reason: reason }
   end
+
+  def track_report
+    RabbitmqBus.send_to_bus('metrics', "report,category=#{category},type=#{reportable_type} count=1")
```
Comment (hennevogel) [inline src/api/app/models/report.rb:43]: Maybe also track the number of sibling reports?

**Review by hennevogel — CHANGES_REQUESTED** [review]

**Review by krauselukas — DISMISSED** [review]

File: src/api/app/models/decision.rb (hunk 2 above)
Comment (dmarcoux) [reply src/api/app/models/decision.rb:33]: Yeah, I will take this out and put it in a function. I was somewhat divided about this when submitting the PR.

**Review by hennevogel — APPROVED** [review]

**Pull request merged**